subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -1834,16 +1834,15 @@ uint8_t ull_adv_sync_pdu_set_clear(struct lll_adv_sync *lll_sync,
 
 			adv = HDR_LLL2ULL(lll_sync->adv);
 
-			adi_pdu->sid = adv->sid;
-
 			/* The DID for a specific SID shall be unique.
 			 */
 			const uint16_t did =
-				ull_adv_aux_did_next_unique_get(adv->sid);
-			adi_pdu->did = sys_cpu_to_le16(did);
+				sys_cpu_to_le16(ull_adv_aux_did_next_unique_get(adv->sid));
+			adi_pdu->did_sid_packed[0] = (did & 0xFF);
+			adi_pdu->did_sid_packed[1] = ((did >> 8) & 0x0F) + (adv->sid << 4);
```

Comment:
There here seems to have slipped through the introduction of the setter macros.

Reply:
Yes that one slipped through - fixed now